packages/launchpad/src/setup/InstallDependencies.vue

```diff
@@ -6,6 +6,7 @@
     :next-fn="confirmInstalled"
     class="max-w-640px relative"
     :main-button-variant="canNavigateForward ? 'primary' : 'pending'"
+    :skip-fn="!canNavigateForward && intervalQueryTrigger.isActive.value ? confirmInstalled : undefined"
```

Comment:
Shouldn't we just let them skip straight away? Why do we wait 18s (that's really long). Or am I reading this logic incorrectly?

Reply:
It shows the skip immediately - but is gonna be hidden after `18s`

![image](https://user-images.githubusercontent.com/15656860/159934095-42291f9c-6ea7-495c-9062-3ebb1bfaa6c9.png)

![image](https://user-images.githubusercontent.com/15656860/159934239-78f9b2d0-6427-4925-9732-afab2af9a9c2.png)

Reply:
Looks like it's 180 seconds... but given that we're waiting for them to install dependencies, more time there is better. 

If we're giving them the ability to skip now, I'm not sure why we need to stop the interval at all until they take some sort of action (they installed the dependencies, they go back, or they skip). I guess if someone were to park at this view, it's nice that we stop eventually as a sanity check?

But if that is the case, it also feels weird to have the "Waiting..." button look active when it's technically not doing anything until Refresh is selected. It seems like we could repurpose the "Waiting" button, enable it when paused and present a different message, and thus keep "Skip" always accessible. This is definitely out of scope for this PR, but might be worth some thought at some point.

Reply:
This feature feels wrong - a skip button that goes away after 18 seconds? What kind of button randomly changes after some interval?

I don't see "refresh" in the designs. Where is this requirement specified?

Code seems fine but unless I missed something the feature seems incorrect.